src/posix/platform/uart.cpp

```diff
@@ -299,10 +299,27 @@ void platformUartProcess(const fd_set *aReadFdSet, const fd_set *aWriteFdSet, co
     {
 #if OPENTHREAD_POSIX_CONFIG_DAEMON_ENABLE
         // Don't die on SIGPIPE
+#if !defined(MSG_NOSIGNAL)
```

Comment:
`#ifdef` `#ifndef`?

Reply:
Is there a style preference for #ifdef/#ifndef when there's only a single check?  I somewhat prefer "defined" because it's usable in ALL situations, regardless of how many things you're checking, but I don't care, especially if there's a documented preference in OpenThread.